File: enrich.js
const fs = require('fs');
const content = fs.readFileSync('src/data/vocabulary.ts', 'utf-8');

const match = content.match(/export const HSK_1_WORDS_LIST = \[([\s\S]*?)\];/);
if (match) {
    console.log("Found words block, length:", match[1].length);
} else {
    console.log("Not found");
}

File: enrich.py
import re

with open('src/data/vocabulary.ts', 'r') as f:
    content = f.read()

match = re.search(r'export const HSK_1_WORDS_LIST = \[(.*?)\];', content, re.DOTALL)
if match:
    words_str = match.group(1)
    print("Found words block")
else:
    print("Not found")
